include/dts-functions.sh

```diff
@@ -1518,6 +1520,18 @@ main_menu_options(){
         return 0
       fi
 
+      echo -n "DTS Extensions: "
```

Comment:
```suggestion
      echo "Your credentials give access to:"
      echo -n "DTS Extensions: "
```

And remove `      echo "Your credentials give access to:"` couple lines below

Reply:
e2886deb4fbc01d9e7bc97d72b6f4fed826c78a8